src/util/apiUtils.js

```diff
@@ -82,14 +82,24 @@ export const parseApiResponse = requestUrl => ([response, body]) => {
 		};
 	}
 	try {
-		value = JSON.parse(body);
-		if (value && value.problem) {
-			value = formatApiError(new Error(`API problem: ${value.problem}: ${value.details}`));
+		if (response.statusCode === 204) { //NoContent response type
+			value = {
+				success: true
+			};
```

Comment:
will `NoContent` _always_ mean that something was successful? I'm assuming this is expected for certain `POST` responses? I'm wondering if it might make more sense to put this `success: true` key on `meta` rather than `value`.

I haven't used this kind of response before (obviously), so we should make a long-term design decision now